runtime/lua/vim/fcnotify.lua

```diff
@@ -0,0 +1,225 @@
+-- Create a file watcher, each watcher is identified by the name of the file that it
+-- watches.
+
+local uv = vim.loop
+local Watcher = {}
+Watcher.__index = Watcher
+local w = {}
+local WatcherList = {}
+local check_handle = nil
+
+-- Callback for the check handle, checks if there are pending notifications
+-- for any watcher, and handles them as per the value of the `fcnotify`
+-- option.
+local function check_notifications()
+  for _, watcher in pairs(WatcherList) do
+    if watcher.pending_notifs == true then
+      vim.api.nvim_command('checktime '..watcher.bufnr)
```

Comment:
is this still the best way to refresh the page?

Perhaps you can add a new `vim.checktime` function to do this without having to use `nvim_command`. You can expose the C function as a lua function.

Reply:
I think I should make another Pull Request for this.